Tests/FoundationEssentialsTests/DataIOTests.swift

```diff
@@ -238,7 +238,7 @@ class DataIOTests : XCTestCase {
     }
     
     func test_zeroSizeFile() throws {
-        #if !os(Linux)
+        #if !os(Linux) && !os(Android)
```

Comment:
We might have discussed this before, but is there any semantic difference between the `os(Android)` used here in the test files, and the `canImport(Android)` used in the implementation?

Reply:
None, I simply followed the convention I've seen elsewhere in the toolchain source of using `canImport(randomOS)` when invoking platform-specific APIs and `os(randomOS)` when separating actual OS differences. Since this `/proc` behavior is in the OS, I used `os(Android)` here. I did not go changing conditional compilation checks for other non-Android OS's that may be doing this wrong to use the same convention though.

Reply:
Ah ok, sounds good - as long as we have a reasoning for it and they're semantically equivalent, this seems fine to me